py_gtfs_rt_ingestion/ingest.py

```diff
@@ -90,13 +91,16 @@ def main(files: list[str]) -> None:
 
     if len(failed_ingestion) > 0:
         logging.warning("Unable to process %d files" % len(failed_ingestion))
+        move_3s_objects(failed_ingestion, os.environ['ingest_bucket'], os.environ['error_bucket'])
 
     logging.info("Writing Table to %s" % os.environ['OUTPUT_DIR'])
     pq.write_to_dataset(
         pa_table,
         root_path=os.environ['OUTPUT_DIR'],
         partition_cols=['year','month','day','hour']
     )
+    files_to_archive = list(set(files) - set(failed_ingestion))
+    move_3s_objects(files_to_archive, os.environ['ingest_bucket'], os.environ['archive_bucket'])
```

Comment:
How do we want to implement pulling environmental variables?

Reply:
i think its best to pull them inside of a try / except block that will check for KeyErrors and re-raise if they happen.

Reply:
https://github.com/mbta/lamp/blob/cb6c9032823dcc0d98cd144eb40865a25a801f3e/py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/batcher.py#L32-L40

Reply:
Incoroprated